changes/5078.removed

```diff
@@ -0,0 +1 @@
+Removes startplugin Nautobot management command.
```

Comment:
```suggestion
Removed `nautobot-server startplugin` management command.
```

Would probably be good to elaborate in this or another changelog to point users to the cookiecutter in its place.